prov/gni/include/gnix_mr.h

```diff
@@ -41,6 +41,12 @@
 #include "gnix_priv.h"
 #include "gnix_mr_cache.h"
 
+#define HAVE_UDREG 1
```

Comment:
A goof up. I put it there so my editor wouldn't automatically turn off auto-complete for the udreg sections of code.